pages/blog/material-ui-v4-is-out.md

```diff
@@ -0,0 +1,309 @@
+---
+description: Material-UI v4 is out 🎉
+---
+
+# Material-UI v4 is out 🎉
+
+> React components for faster and easier web development. Build your own design system, or start with Material Design.
+
+Material-UI v4 has finally arrived. We are so excited about this release, as it defines better foundations for the UI components. Thank you to everyone, especially to [the team](/discover-more/team/), and to everyone who's contributed code, issue triage, and support. **Thank you.**
+
+✨✨✨ See the **[4.0.0 Release Note](https://github.com/mui-org/material-ui/releases/tag/v4.0.0)** on GitHub. ✨✨✨
+
+![TypeScript switch](/static/blog/material-ui-v4-is-out/banner.png)
+
+<p class="blog-description">https://material-ui.com/</p>
+
+## Summary
+
+- [High-level goals for v4](#high-level-goals-for-v4)
+- [What's new?](#whats-new)
+- [What's next?](#whats-next)
+- [Premium themes store](#premium-themes-store)
+
+## High-level goals for v4
+
+Material-UI v1 was [released](https://medium.com/material-ui/material-ui-v1-is-out-e73ce13463eb) one year ago. This is our most important major release since then. For those unfamiliar with the history of the project, we didn't release a v2, and v3 only introduced a slight browsers support change.
```

Comment:
```suggestion
Material-UI v1 was [released](https://medium.com/material-ui/material-ui-v1-is-out-e73ce13463eb) one year ago. Version 4 is our most important major release since then. For those unfamiliar with the history of the project, we didn't release a v2, and v3 only introduced a slight browsers support change.
```